010-OFEP-flagd-project-charter.md

```diff
@@ -0,0 +1,41 @@
+## Make flagD a full sub-project of OpenFeature
+
+## State: PENDING REVIEW
+
+
+This OFEP proposes to make flagD a full subproject of OpenFeature. The project will get a dedicated website, documentation, roadmap and charter. 
+
+## Background
+
+flagD started as small reference implementation for a developer-focussed feature evaluation mechanism, enabling OpenFeature SDK users to experiment with an end-to-end solution. Over time the project has grown in functionality and become stable and is now also used in production settings. 
+
+In the past there was also some misunderstanding of the relationship of OpenFeature and flagD. 
+
+Therefore, the project should become a full sub-project of open feature. 
+
+## Proposal
+
+flagD will become a dedicated sub project of the OpenFeature project. Documentation for flagD and all examples for OpenFeature will be moved to a dedicted space under the newly created domain ``openfeature.dev``.
```

Comment:
Just to clarify, is the intention that it would live in a different sub-domain - like flagd.openfeature.dev - or just a separate area of the https://openfeature.dev site?

Reply:
Yes, this was my mistake. It should be flagd.dev